src/Classes/ServiceAPI/MyRadio_TracklistItem.php

```diff
@@ -187,6 +187,12 @@ public static function getTracklistForTime($start, $end = null, $include_playout
             unset($data['digitised']);
             unset($data['deletelink']);
             unset($data['trackno']);
+            unset($data['intro']);
+
+            //for manual SIS entries
+            if (!isset($data['trackid'])) {
+                $data['trackid'] = "SIS Manual";
```

Comment:
Hmm, I guess, but I wanted to make it obvious. It's only used in the fullTrackList / csv from what I can see.